### PR TITLE
chore: reduce test duration

### PR DESCRIPTION
**Pull request by AjimenezDCL**
Reduce the absolute time a test run job in the CI takes.

### CODE REVIEW

**Comment by AjimenezDCL** [comment]
Results at the beginning of the PR: 505 secs
[results.txt](https://github.com/decentraland/explorer/files/5607632/results.txt)

**Comment by AjimenezDCL** [comment]
Results after reducing Wearables catalog: 403 secs
[results.txt](https://github.com/decentraland/explorer/files/5608309/results.txt)

**Review by BrianAmadori — APPROVED** [review]
LGTM!

**Comment by pravusjif** [comment]
https://github.com/decentraland/explorer/pull/1722/commits/288b1ab86ec6023578722b8b5fd88886c79dee29
showed a 14 seconds gain when testing in local

**Review by BrianAmadori — APPROVED** [review]
Approving this again. Great job!

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatHUDController.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 public class ChatHUDController : IDisposable
 {
-    public const int MAX_CHAT_ENTRIES = 100;
+    public static int MAX_CHAT_ENTRIES = 100;
```
Comment (pbosio) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatHUDController.cs]:
```suggestion
    public static int MAX_CHAT_ENTRIES { internal set; get; } = 100;
```
If we are going to transform this into a variable, can we modify it to minimize any risk of changing it value by mistake?

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/Tests/ChatHUDShould.cs [hunk 2]
```diff
@@ -39,6 +39,9 @@ public void SubmitMessageProperly()
     [Test]
     public void TrimWhenTooMuchMessagesAreInView()
     {
+        int cacheMaxEntries = ChatHUDController.MAX_CHAT_ENTRIES;
+        int newMaxEntries = 10;
```
Comment (pbosio) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/Tests/ChatHUDShould.cs]:
```suggestion
        const int newMaxEntries = 10;
```

**Review by pbosio — APPROVED** [review]

**Pull request merged**